src/settings.py

```diff
@@ -215,8 +215,8 @@ class GlobalSettings:
     gnn_option_policy_shooting_variance = 0.1
     gnn_option_policy_shooting_max_samples = 100
 
-    # parameters for GNN metacontroller approach
-    gnn_metacontroller_max_samples = 100
+    # parameters for metacontroller approaches
```

Comment:
looks like you already did but might as well:
git grep -i "gnn meta"
git grep -i "gnnmeta"
git grep -i "gnn_meta"
to make sure nowhere else needs to change